app/controllers/api/v1/conversations/messages_controller.rb

```diff
@@ -0,0 +1,42 @@
+module Api
+  module V1
+    module Conversations
+      class MessagesController < Api::V1::ApiController
+        include Pagy::Backend
+
+        helper_method :conversation, :user
+
+        def create
+          @message = conversation.messages
+                                 .create!(message_params)
+          @message.update!(user_id: user.id)
+          @message.notify
+          render 'api/v1/messages/show'
+        end
+
+        def index
+          @pagy, @messages = pagy(conversation.messages.order('id DESC'))
```

Comment:
You can use `order(id: :desc)`. Also, I would move this to a scope, for example: `order_desc`